srv/components/misc_pkgs/swupgrade/repo/iso/mount.sls

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+{% macro repo_mounted(release, source, mount_dir) %}
+
+sw_upgrade_repo_iso_mounted_{{ release }}:
```

Comment:
We need to check the number of mounts at some point.
This is not the place for sure, but do keep in mind that we would be maintaining 4 ISO versions max at any given point:
- Factory version
- One old version
- Current verion
- Upgrade version
On successful upgrade the old current version becomes old version and the old version would be removed.

Reply:
@dmitrydb please create a task for that with description from that comment. Need to discuss more later.